.ci/setup.sh

```diff
@@ -100,7 +101,10 @@ enable_nested_virtualization() {
 install_kata() {
 	if [ "${INSTALL_KATA}" == "yes" ]; then
 		echo "Install Kata sources"
-		bash -f ${cidir}/install_kata.sh
+		if [ -n "${TEE_TYPE}" ]; then
+			echo "Using with TEE type: ${TEE_TYPE}"
+		fi
+		bash -f ${cidir}/install_kata.sh "${TEE_TYPE}"
```

Comment:
Hi @arronwy ! Let me ask you one last thing... here I think you should continue passing the `tag` and remove https://github.com/kata-containers/tests/pull/4668/commits/19aa47e84581228edc0b7d81fc6720c833793fe8#diff-488b8054ced3f482b39437df10e2189b4acef7470bfb5d0ce4b70a7cff5cbbedR13 . The value of `TEE` is exported and the `install_kata.sh` handles it; so no need to pass a script parameter.

Reply:
@wainersm , thanks, fixed now.